locales/en-US/app.ftl

```diff
@@ -413,6 +413,12 @@ MenuButtons--metaInfo--attempting-resymbolicate = Attempting to re-symbolicate p
 MenuButtons--metaInfo--currently-symbolicating = Currently symbolicating profile
 MenuButtons--metaInfo--cpu = CPU:
 MenuButtons--metaInfo--main-memory = Main memory:
+MenuButtons--moreInfo-button = More
+
+## MoreInfo panel
+## These strings are used in the panel containing additional information
```

Comment:
If the comment applies only to the following string, it should only use one #.

Also, the existing group comment (`## MetaInfo Panel`) applied to all the strings after (line 422-> here), now this replaces it, so I would remove the `MoreInfo panel` completely?